infrastructure/zk/src/init.ts

```diff
@@ -138,6 +133,67 @@ export async function submoduleUpdate() {
     await utils.exec('git submodule update');
 }
 
+async function configMode(validiumMode: boolean) {
+    let envFileContent = fs.readFileSync(process.env.ENV_FILE!).toString();
+    envFileContent += `VALIDIUM_MODE=${validiumMode}\n`;
+    fs.writeFileSync(process.env.ENV_FILE!, envFileContent);
+    await announced(`Initializing in ${validiumMode ? 'Validium mode' : 'Roll-up mode'}`);
+
+    const chainPath = 'etc/env/base/chain.toml';
+    const modeConstantValues = {
+        pubdata_overhead_part: validiumMode ? 0.0 : 1.0,
+        batch_overhead_l1_gas: validiumMode ? 1000000 : 800000,
+        max_pubdata_per_batch: validiumMode ? 1000000000000 : 100000
+    };
+
+    let chainContent = fs.readFileSync(chainPath, 'utf-8');
+    const lines = chainContent.split('\n');
+
+    for (let i = 0; i < lines.length; i++) {
+        const line = lines[i];
+        for (const [key, value] of Object.entries(modeConstantValues)) {
+            if (line.includes(`${key}=`)) {
+                lines[i] = `${key}=${value}`;
+                break;
+            }
+        }
+    }
+    chainContent = lines.join('\n');
+    fs.writeFileSync(chainPath, chainContent);
+
+    await announced(`The parameters have been updated in the ${chainPath} file.`);
+
+    const ethSenderPath = 'etc/env/base/eth_sender.toml';
+    const enforcedGasPrice = 'internal_enforced_l1_gas_price';
```

Comment:
Let's keep names consistent

```suggestion
    const internalEnforcedL1GasPrice = 'internal_enforced_l1_gas_price';
```